examples/bread/luci.snapshot.json

```diff
@@ -1,77 +1,59 @@
 {
-  "name": "bakery",
+  "name": "integration_test",
   "targets": [
     {
-      "path": "//farm:water",
+      "path": "//regular_integration_tests:text_editing",
       "buildTarget": {
-        "name": "water",
+        "name": "text_editing",
         "agentProfiles": [
-          "linux"
+          "linux", 
+          "macos", 
+          "ios", 
+          "android"
```

Comment:
Listing multiple `agentProfiles` means the target will run on any one of them. If you want to run once for every operating system, then you will generate multiple targets.